<launch>

    <!-- <arg name="enable_debug" /> -->
    <arg name="enable_debug"                default="false"/>
    <arg name="is_ground_vehicle"           default="true"/>
    <group ns="OHRAD_X3">
    <node pkg="nearness_control" type="nearness_control_3d_node" name="nearness_control_3d_node" output="screen">

        <!-- SUBSCRIBED TOPICS -->
        <!-- <remap from="nearness_controller_3d/points" to="ouster_points"/> -->

        <!-- Enable switches -->
        <!-- <param name="enable_debug" type="bool" value="$(arg enable_debug)"/> -->

        <!-- Sensor Parameters -->

    </node>

    <include file="$(find nearness_control)/launch/sim/launch_tf2odom.launch"/>

    </group>

    <include file="$(find nearness_control)/launch/sim/launch_joy_teleop_mixer.launch"/>


</launch>
